src/transport/SessionHandle.cpp

```diff
@@ -23,12 +23,59 @@ namespace chip {
 
 using namespace Transport;
 
+using AuthMode          = Access::AuthMode;
 using SubjectDescriptor = Access::SubjectDescriptor;
 
 SubjectDescriptor SessionHandle::GetSubjectDescriptor() const
 {
-    SubjectDescriptor subjectDescriptor = { .fabricIndex = mFabric };
-    // TODO: fill subject descriptor with proper fields
+    SubjectDescriptor subjectDescriptor;
+    if (IsSecure())
+    {
+        if (IsOperationalNodeId(mPeerNodeId))
+        {
+            // Currently ending up here after commissioning (good)
+            subjectDescriptor.authMode    = AuthMode::kCase;
+            subjectDescriptor.subject     = mPeerNodeId;
+            subjectDescriptor.fabricIndex = mFabric; // have fabric here
+            // TODO: add CATs
+            ChipLogDetail(DataManagement, "@@@ #####################################  CASE  %lx  (%u)", subjectDescriptor.subject,
+                          subjectDescriptor.fabricIndex);
+        }
+        else if (IsPAKEKeyId(mPeerNodeId))
+        {
+            // Should end up here during commissioning (but currently not)
+            subjectDescriptor.authMode    = AuthMode::kPase;
+            subjectDescriptor.subject     = mPeerNodeId;
```

Comment:
Need to clearly differentiate whether PeerNodeId is Ephemeral NodeID used for transport tracking purposes over unencrypted session (0xFFFF_FFFE_xxxx_xxxx) or PAKE key id (0xFFFF_FFFB_xxxx_xxxx).
We clearly want to be using the PAKE key id aka `passcodeID` for subject.  Perhaps this can be a `TODO(#issue)` for now, similar to CAT.

Reply:
From an AccessControl perspective (and therefore SubjectDescriptor), anything that isn't authenticated as PASE, CASE, or Group, is not relevant. That is, they exist at the messaging level, but don't need to be represented in the SubjectDescriptor, because there will be no ACLs to check that would ever match such a case. There is already an issue for PASE in SubjectDescriptor, I added it to the TODO.